ethereum/contracts/pyth/Pyth.sol

```diff
@@ -39,6 +39,7 @@ contract Pyth is PythGetters, PythSetters, AbstractPyth {
 
             if(attestation.attestationTime > latestPrice.attestationTime) {
                 setLatestPriceInfo(attestation.priceId, newPriceInfo(attestation));
+                emit PriceUpdate(attestation.priceId, attestation.publishTime);
```

Comment:
Do we need to declare the `PriceUpdate` event anywhere?

Reply:
It's in the solidity sdk.